src/test/kotlin/no/skatteetaten/aurora/gobo/integration/cantus/ImageRegistryServiceBlockingContractTest.kt

```diff
@@ -20,7 +20,10 @@ import org.springframework.cloud.contract.stubrunner.spring.AutoConfigureStubRun
     webEnvironment = SpringBootTest.WebEnvironment.NONE,
     classes = [ApplicationConfig::class, ImageRegistryServiceBlocking::class]
 )
-@AutoConfigureStubRunner(ids = ["no.skatteetaten.aurora:cantus:+:stubs:6568"])
+@AutoConfigureStubRunner(
+    ids = ["no.skatteetaten.aurora:cantus:AOS_3290_post_for_queryparametre-SNAPSHOT:stubs:6568"],
```

Comment:
Dette må vel endres tilbake før merge?

Kan vi legge inn denne kodesnutten i Readme? Hvordan teste med snapshot-stubs?

Reply:
Jeg har noen idéer om hvordan vi løser dette. Tanken er at vi bruker placeholdere to steder i konfigurasjonen, det ene er "repositories" / "snapshots" og det andre er "+"

Dette bør også forklares i readme ja.

Reply:
Tar å bytte tilbake til slik det er i master, det er laget en egen oppgave på dette.